### ISSUE TITLE
Use strict semantic versioning

### ISSUE THREAD

**Issue by nounder** [nats-io/nats.js#50]
### Proposed change

Use strict MAJOR.MINOR.PATCH semantic versioning. Currently a `-N` suffix is added to a version.

### Use case

I'm not sure what's use of the suffix for this library but it causes problems in Deno when downloading most recent version of the library:
```
> import * as mod from "jsr:@nats-io/nats-transport-deno"
Uncaught TypeError:
    Could not find version of '@nats-io/nats-transport-deno' that matches specified version constraint '*'
```

I think it would be beneficial for everyone to adhere to commonly used naming conventions to ensure compatibility across diverse JS ecosystem.

**Comment by nounder** [comment]
I believe this would fix #48

**Comment by aricart** [comment]
its not released - so since it doesn't have a release, you have to specify the qualifier, you can find out a version by looking at the versions. - Once there's a real version that will work
